lib/sav_types/utilType.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass, fields
+
+class Null:
```

Comment:
in json file have null and "NoneType". but python doesn't know what is null or None.